var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -115,7 +115,7 @@ class Hiop(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("coinhsl+blas", when="+sparse")
     depends_on("metis", when="+sparse")
 
-    depends_on("ginkgo@1.5.0.glu_experimental", when="+ginkgo")
+    depends_on("ginkgo@1.5.0", when="+ginkgo")
```

Comment:
I believe that HiOp does depend on that specific branch `glu_experimental` of Ginkgo, which still has some complex unreleased code, although a big chunk was released now. For the purpose of this xSDK release, maybe `hiop~ginkgo` should be set

Reply:
Ok. I had assumed `1.5.0.glu_experimental` changes got merged into 1.5.0/develop - so attempted to remove it. Will revert this [after CI is done..]

xsdk doesn't enable hiop+ginkgo..

Reply:
I agree Ginkgo support in HiOp should not be part of this xSDK release.